packages/database/src/migrations/20210510061852-AddPSSSActiveAlertsReport-modifies-data.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const REPORT_CODE = 'PSSS_Active_Alerts';
+
+exports.up = async function (db) {
+  const report = {
+    id: generateId(),
+    code: REPORT_CODE,
+    config: {
+      fetch: {
+        dataGroups: ['PSSS_Alert'],
+        dataElements: ['PSSS_Alert_Syndrome', 'PSSS_Alert_Archived'],
+      },
+      transform: [
+        {
+          // filter active alerts
+          transform: 'filter',
+          where: "eq($row['PSSS_Alert_Archived'], 0) or notExists($row['PSSS_Alert_Archived'])",
```

Comment:
```suggestion
          where: "eq($row['PSSS_Alert_Archived'], 0) or not exists($row['PSSS_Alert_Archived'])",
```
Totally optional, but it's pretty neat that mathjs has such boolean logic built into it so we don't have to implement inverse functions if we don't want 😄